g2p/tests/test_cli.py

```diff
@@ -121,35 +121,43 @@ def test_convert(self):
         )
         error_count = 0
         for tok_option in [["--tok", "--check"], ["--no-tok"]]:
-            for test in langs_to_test:
+            for row in langs_to_test:
+                (in_lang, out_lang, word_to_convert, reference_string, fileline) = row[
+                    :5
+                ]
```

Comment:
There's a bug here. `fileline` is not `row[4]`, but `row[-1]`. Most of the time, that's the same thing, but our test data allow adding an extra column for comments. See `moh.psv` for example. That's why we were using `test[-1]` in the previous code rather than `test[4]`.

So this should probably be
```
(in_lang, out_lang, word_to_convert, reference_string, *_, fileline) = row
```

Reply:
Ah, oops, you're right, I missed that.  I didn't realize the syntax above was possible either, in fact it is what I was looking for in this case but couldn't find (horror of horrors, I assumed that it would be `...` like in JavaScript!)

So another PR will arrive in a second.